CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## x.y.z
 
 ### Added
+- Test `:nth-last-child`
```

Comment:
Thinking about it, it's not just tests being added.  These are new features by virtue of the Symfony CssSelector component.  So these last 3 CHANGELOG entries should in fact read “Support and test …”.  I can change that in a separate PR.